spec/models/work_package/work_package_planning_spec.rb

```diff
@@ -170,36 +170,25 @@
         expect(planning_element).not_to be_valid
 
         expect(planning_element.errors[:project]).to be_present
-        expect(planning_element.errors[:project]).to eq(["can't be blank"])
+        expect(planning_element.errors[:project]).to eq(["can't be blank."])
       end
     end
 
     describe 'parent' do
-      let (:de_message) { 'darf kein Meilenstein sein' }
```

Comment:
With that comment block regarding AR error messages you removed earlier I can see *why* someone decided to add localized specs, but I WTFed at the whole concept / assumption that this originated from. Either way, I'm glad they're gone :D